cmd/lncli/cmd_custom.go

```diff
@@ -51,3 +52,32 @@ func sendCustom(ctx *cli.Context) error {
 
 	return err
 }
+
+var subscribeCustomCommand = cli.Command{
```

Comment:
Not sure how useful this _CLI_ command will be in practice, but don't see much harm in including it /shruggie

Reply:
Yes, its use is doubtful. But because this is generic functionality, there may be an unexpected use for it. In any case, it is useful in development.